src/promptflow-tools/promptflow/tools/yamls/open_source_llm_with_custom_llm.yaml

```diff
@@ -0,0 +1,25 @@
+promptflow_openllm.tool.open_source_llm.OpenSourceLLM.call_with_custom_llm:
+  name: Open Source LLM - Custom LLM
+  description: Tool to use Azure Foundation Models
```

Comment:
Please kindly refer to other tool description and improve this one to be more accurate.